test/import/FSHErrorListener.test.ts

```diff
@@ -1,33 +1,36 @@
 import { loggerSpy } from '../testhelpers/loggerSpy';
 import { importSingleText } from '../testhelpers/importSingleText';
+import { leftAlign } from '../utils/leftAlign';
 
 describe('FSHErrorListener', () => {
   beforeEach(() => loggerSpy.reset());
 
   it('should not log any errors for valid FSH', () => {
-    const input = `
+    const input = leftAlign(`
     Profile: MyPatient
     Parent: Patient
     * active = true
-    `;
+    `);
     importSingleText(input, 'MyPatient.fsh');
     expect(loggerSpy.getAllMessages('error')).toBeEmpty();
   });
 
   it('should report mismatched input errors from antlr', () => {
-    const input = `
+    const input = leftAlign(
+      leftAlign(`
```

Comment:
I don't think you meant to call `leftAlign` twice here.

Reply:
I did not! Thank you for catching this.